pipeline-modules/deployment-service/gcp/basic-gke-regional.yaml

```diff
@@ -163,6 +165,9 @@ template: |
     - '-c'
     - |
       export PATH="{{workspace}}/bin:$$PATH";
+      export PATH="{{ common_scripts_path }}:$$PATH";
+      install-jq.sh      
```

Comment:
if you are just going to call the one script - you can do:
{{ common_scripts_path }}/install-jq.sh
or collapse line 167/168 into on export PATH= stmt

Reply:
Update the {{ common_scripts_path }}/install-jq.sh